code/modules/projectiles/projectile/special.dm

```diff
@@ -387,3 +387,206 @@
 		var/obj/effect/ebeam/B = b
 		animate(B, alpha = 0, time = 32)
 	return ..()
+
+/obj/item/projectile/hallucination
```

Comment:
so far i've added bolt of change and bolt of death since we have hallucinations that fit those effects really well